content/developer/reference/frontend/owl_components.rst

```diff
@@ -150,13 +150,144 @@ checkboxes or datepickers. This page explains how to use these generic component
 
    * - Technical Name
      - Short Description
+   * - :ref:`ActionSwiper <frontend/owl/actionswiper>`
+     - a swiper component to perform actions on touch swipe
    * - :ref:`CheckBox <frontend/owl/checkbox>`
      - a simple checkbox component with a label next to it
    * - :ref:`Dropdown <frontend/owl/dropdown>`
      - full-featured dropdown
    * - :ref:`Pager <frontend/pager>`
      - a small component to handle pagination
 
+.. _frontend/owl/actionswiper:
+
+ActionSwiper
+------------
+
+.. note:: This component is a mobile feature, only supported in the Enterprise version of Odoo.
+
+Location
+~~~~~~~~
+
+`@web_enterprise/core/action_swiper/action_swiper`
+
+Description
+~~~~~~~~~~~
+
+This is a component that can perform actions when an element is swiped
+horizontally. The swiper is wrapping a target element to add actions to it. 
+The action is executed once the user has released the swiper passed
+half of its width.
+
+.. code-block:: xml
+
+  <ActionSwiper onLeftSwipe="Object" onRightSwipe="Object">
```

Comment:
maybe use some more realistic function name: `doSomething` and `doSomethingElse` instead of Object

Reply:
ok, i did not know the API for this component. Note that it is not intuitive, in my opinion. It should be something like `onLeftSwipe` (function), `onRightSwipe` (function), `leftColor`, `rightColor`, ... (with defaults if possible)

Reply:
Should I keep the current documentation until the API is changed in the future?